firmware/controllers/actuators/electronic_throttle.cpp

```diff
@@ -269,6 +274,29 @@ static percent_t currentEtbDuty;
 
 		etb1.dcMotor.Set(ETB_PERCENT_TO_DUTY(currentEtbDuty));
 
+	   	float etbError = absF(targetPosition - actualThrottlePosition); // Used for I-Term limiter and Error-detection
+
+
+        // ETB Deviation monitoring
+        if(etbError < 3 ){
+          start_time = chVTGetSystemTimeX(); //Updates timestamp as long as ETB-Error stays within 3% deviation
+        }
+      else {
+        }        // ETB Deviation outside limit detected, set timestamp and monitor it for two seconds
+        if(chVTIsSystemTimeWithinX(start_time, start_time + TIME_MS2I(2000))) {
+        	  //Less than two seconds, system OK: do nothing
+            }
+       else {
+         if(rpm > 600){  //We dont want or need any of this to execute if engine are not running
+                         // It would make a huge issue while tuning ETB
+        	 etbErrorCondition = true;
+        	   warning(OBD_Throttle_Actuator_Control_Range_Performance_Bank_1, "ETB Deviation Error"); //Set Error-code
```

Comment:
A better error message might be "Electronic throttle deviation limits exceeded"

Reply:
I have "Electronic_Throttle_Position_Control_Error" in my code, but I used the one you already had in this file to not edit more files for this PR

Reply:
I mean the second parameter, currently `"ETB Deviation Error"`

Reply:
Ah, ok. I can change that.